v2/pkg/protocols/common/utils/vardump/dump.go

```diff
@@ -0,0 +1,34 @@
+package vardump
+
+import (
+	"strings"
+
+	"github.com/projectdiscovery/gologger"
+	"github.com/projectdiscovery/nuclei/v2/pkg/types"
+)
+
+// Variables writes the truncated dump of variables on the stderr
+// in a formatted key-value manner using gologger.
+//
+// The values are truncated to return 50 characters from start and end.
+func Variables(data map[string]interface{}) {
+	var counter int
+
+	builder := &strings.Builder{}
+	for k, v := range data {
+		valueString := types.ToString(v)
+
+		counter++
+		if len(valueString) > 50 {
```

Comment:
I would understand the truncation if the whole content would be displayed through an increased log level. If that's not the case, I would prefer seeing the whole content.

Reply:
The full content in this case is redundant as it's the HTTP response and is already shown when running debug, so it will just be same content. Instead here we are more interested in seeing what variables are available